src/new-client/src/services/currencyPairs/currencyPairs.ts

```diff
@@ -41,13 +41,14 @@ export const currencyPairUpdates$ = getStream$<RawCurrencyPairUpdates>(
 export const [useCurrencyPairs, currencyPairs$] = bind(
   currencyPairUpdates$.pipe(
     scan((acc, { updateType, currencyPair }) => {
+      const result = { ...acc }
```

Comment:
It's not causing a real problem right now because we never unsubscribe/resubscribe to this endpoint. However, the problem was coming up with the tests, because since `scan` was mutating the `seed` object, then [this test](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/1817/files#diff-9f2a03f10fee0cef0dc7ab30b59f019ce9f1c2a64d475f782840425b5954cedbR135) was not passing b/c the previous test was setting that value and it was still around. So, yeah, thanks to the tests we've detected a malfunctioning service.